### PR TITLE
Update legacy-pricing.mdx

### PR DESCRIPTION
**Pull request by lizokm**
Correct mistake in AM2 pricing docs (performance unit pricing)

## DESCRIBE YOUR PR
*Tell us what you're changing and why. If your PR **resolves an issue**, please link it so it closes automatically.*

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ X ] Urgent deadline ASAP

### CODE REVIEW

File: docs/pricing/legacy-pricing.mdx [hunk 1]
```diff
@@ -72,10 +72,10 @@ Performance Units can be used towards multiple Performance event types, dependin
 | >4.5M-5M | $0.000046 | $0.000068 | $0.000092 | $0.0000169 |
 | >5M-5.5M | $0.000039 | $0.000060 | $0.000074 | $0.000120 |
 | >5.5M-10M | $0.000039 | $0.000051 | $0.000074 | $0.000096 |
-| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000096 |
+| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000053 |
```
Comment (lizokm) [inline docs/pricing/legacy-pricing.mdx:75]:
```suggestion
| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.0000530 |
```

File: docs/pricing/legacy-pricing.mdx [hunk 2]
```diff
@@ -72,10 +72,10 @@ Performance Units can be used towards multiple Performance event types, dependin
 | >4.5M-5M | $0.000046 | $0.000068 | $0.000092 | $0.0000169 |
 | >5M-5.5M | $0.000039 | $0.000060 | $0.000074 | $0.000120 |
 | >5.5M-10M | $0.000039 | $0.000051 | $0.000074 | $0.000096 |
-| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000096 |
+| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.0000530 |
```
Comment (lizokm) [inline docs/pricing/legacy-pricing.mdx:75]:
```suggestion
| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000053 |
```

**Review by brianthi — APPROVED** [review]

File: docs/pricing/legacy-pricing.mdx [hunk 3]
```diff
@@ -72,10 +72,10 @@ Performance Units can be used towards multiple Performance event types, dependin
 | >4.5M-5M | $0.000046 | $0.000068 | $0.000092 | $0.0000169 |
 | >5M-5.5M | $0.000039 | $0.000060 | $0.000074 | $0.000120 |
 | >5.5M-10M | $0.000039 | $0.000051 | $0.000074 | $0.000096 |
-| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000096 |
+| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000053 |
 | >15M-100M | $0.000027 | $0.000036 | $0.000040 | $0.000053 |
-| >100M-150M | $0.000019 | $0.000036 | $0.000032 | $0.000053 |
-| >150M-200M | $0.000019 | $0.000021 | $0.000032 | $0.000021 |
+| >100M-150M | $0.000019 | $0.000036 | $0.000032 | $0.000042 |
```
Comment (brianthi) [inline docs/pricing/legacy-pricing.mdx:77]: The OD value here should be $0.000053

File: docs/pricing/legacy-pricing.mdx (hunk 1 above)
Comment (brianthi) [inline docs/pricing/legacy-pricing.mdx:75]: Should be $0.000096

File: docs/pricing/legacy-pricing.mdx [hunk 4]
```diff
@@ -72,10 +72,10 @@ Performance Units can be used towards multiple Performance event types, dependin
 | >4.5M-5M | $0.000046 | $0.000068 | $0.000092 | $0.0000169 |
```
Comment (brianthi) [inline docs/pricing/legacy-pricing.mdx:72]: $0.0000169 should be $0.000169. Has one too many zeros.

File: docs/pricing/legacy-pricing.mdx (hunk 1 above)
Comment (lizokm) [inline docs/pricing/legacy-pricing.mdx:75]:
```suggestion
| >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000096 |
```

File: docs/pricing/legacy-pricing.mdx [hunk 5]
```diff
@@ -74,8 +74,8 @@ Performance Units can be used towards multiple Performance event types, dependin
 | >5.5M-10M | $0.000039 | $0.000051 | $0.000074 | $0.000096 |
 | >10M-15M | $0.000027 | $0.000051 | $0.000040 | $0.000096 |
 | >15M-100M | $0.000027 | $0.000036 | $0.000040 | $0.000053 |
-| >100M-150M | $0.000019 | $0.000036 | $0.000032 | $0.000053 |
-| >150M-200M | $0.000019 | $0.000021 | $0.000032 | $0.000021 |
+| >100M-150M | $0.000019 | $0.000036 | $0.000032 | $0.000042 |
```
Comment (lizokm) [inline docs/pricing/legacy-pricing.mdx:77]:
```suggestion
| >100M-150M | $0.000019 | $0.000036 | $0.000032 | $0.000053 |
```

**Pull request merged**